icu4c/source/test/intltest/plurults.cpp

```diff
@@ -466,10 +466,10 @@ void PluralRulesTest::testGetFixedDecimalSamples() {
 
     FixedDecimal values[1000];
     for (int32_t i = 0; U_SUCCESS(status) && i < numLocales; ++i) {
-        if (uprv_strcmp(locales[i].getLanguage(), "fr") == 0 &&
-                logKnownIssue("21322", "PluralRules::getSamples cannot distinguish 1e5 from 100000")) {
-            continue;
-        }
+        //if (uprv_strcmp(locales[i].getLanguage(), "fr") == 0 &&
+        //        logKnownIssue("21322", "PluralRules::getSamples cannot distinguish 1e5 from 100000")) {
+        //    continue;
+        //}
```

Comment:
Remove comment?

Reply:
See above. Since the ICU ticket is not resolved, I was worried that the problem may depend on values that happen to be in samples. I wanted to leave this there in case we need it again.